src/bp/ui-studio/src/web/views/FlowBuilder/diagram/index.tsx

```diff
@@ -680,51 +696,62 @@ class Diagram extends Component<Props> {
-            placeholder={lang.tr('studio.flow.filterNodes')}
-            onChange={value => this.props.handleFilterChanged({ target: { value } })}
-          />
-        </div>
-        <div
-          id="diagramContainer"
-          ref={ref => (this.diagramContainer = ref)}
-          tabIndex={1}
-          style={{ outline: 'none', width: '100%', height: '100%' }}
-          onContextMenu={this.handleContextMenu}
-          onDrop={this.handleToolDropped}
-          onDragOver={event => event.preventDefault()}
+      <Fragment>
+        <MainLayout.Wrapper
+          className={cx({
+            'emulator-open': this.props.emulatorOpen
+          })}
         >
-          <div className={style.floatingInfo}>{this.renderCatchAllInfo()}</div>
-
-          <DiagramWidget
-            ref={w => (this.diagramWidget = w)}
-            deleteKeys={[]}
-            diagramEngine={this.diagramEngine}
-            inverseZoom
-          />
-          <ZoomToolbar />
-          {canAdd && <NodeToolbar />}
-          <TriggerEditor
-            node={this.state.currentTriggerNode}
-            isOpen={this.state.isTriggerEditOpen}
-            diagramEngine={this.diagramEngine}
-            toggle={() => this.setState({ isTriggerEditOpen: !this.state.isTriggerEditOpen })}
-          />
-        </div>
-      </MainLayout.Wrapper>
+          <WorkflowToolbar />
+
+          <div className={style.searchWrapper}>
+            <SearchBar
+              id="input-highlight-name"
+              className={style.noPadding}
+              ref={this.searchRef}
+              onBlur={this.props.hideSearch}
+              value={this.props.highlightFilter}
+              placeholder={lang.tr('studio.flow.filterNodes')}
+              onChange={value => this.props.handleFilterChanged({ target: { value } })}
+            />
+          </div>
+          <div
+            id="diagramContainer"
+            ref={ref => (this.diagramContainer = ref)}
+            tabIndex={1}
+            style={{ outline: 'none', width: '100%', height: '100%' }}
+            onContextMenu={this.handleContextMenu}
+            onDrop={this.handleToolDropped}
+            onDragOver={event => event.preventDefault()}
+          >
+            <div className={style.floatingInfo}>{this.renderCatchAllInfo()}</div>
+
+            <DiagramWidget
+              ref={w => (this.diagramWidget = w)}
+              deleteKeys={[]}
+              diagramEngine={this.diagramEngine}
+              inverseZoom
+            />
+            <ZoomToolbar />
+            {canAdd && <NodeToolbar />}
+            <TriggerEditor
+              node={this.state.currentTriggerNode}
+              isOpen={this.state.isTriggerEditOpen}
+              diagramEngine={this.diagramEngine}
+              toggle={() => this.setState({ isTriggerEditOpen: !this.state.isTriggerEditOpen })}
+            />
+          </div>
+        </MainLayout.Wrapper>
+
+        {/* <Forms
```

Comment:
nicely commented out,leftover ?

Reply:
Intended, I used the say form to change the Form implementation, but I don't want to use it for the Say Something node yet. I'm actually using this form in my next PR which implements the Execute node